clients/ui/scripts/deploy_kind_cluster.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Depending on the system, it's not guaranteed bash will be there, I'd recommend the shebang below, it's pretty much universal.

```suggestion
#!/usr/bin/env bash
```